src/services/dashboardPermmisionsService.ts

```diff
@@ -7,10 +8,11 @@ class DashboardPermissionsService {
     tableName = Tables.dashboardsPermissions;
 
     async addDashboardPermission(dashboardPermissions: DashboardPermissions) {
+		const id = uuid();
         const query = `
         INSERT INTO ${DATABASE_NAME}."${this.tableName}"
-        ("dashboardId", "username")
-        VALUES ('${dashboardPermissions.dashboardId}', '${dashboardPermissions.username}')
+        ("permissionId", "dashboardId", "username")
+        VALUES ('${id}','${dashboardPermissions.dashboardId}', '${dashboardPermissions.username}')
```

Comment:
is should be automatic serial

Reply:
if we want automatic uuid from what I searched up on the internet you need an extension for postgres to do that
It's probably not too hard but I currently dont think it's worth the time if it does get annoying - if you want I can either still do that / add a todo for this